### ISSUE TITLE
Inscrição Estadual para Pessoa Física?

### ISSUE THREAD

**Issue by casaud** [deivisonarthur/OSC-Magento-Brasil-6-Pro#82]
Olá! 

Primeiro gostaria de agradecer pelo belíssimo módulo.

Segundo, é que está aparecendo o campo IE (inscrição estadual) na hora de criar uma conta para a pessoa física. Tem como deixar só para a jurídica?

Abraços!

**Comment by rafaelstz** [comment]
O correto é mudar a label de IE para RG ao clicar em PF. Está aparecendo algum erro no log?

**Comment by casaud** [comment]
O campo RG/CNPJ aparece normalmente.
O IE não oculta quando seleciono o PF
Uso o OSC6
Nenhum erro no log.

**Comment by rafaelstz** [comment]
Você consegue criar facilmente ajustes de acordo com sua necessidade nesse arquivo do link a partir da linha **228** aonde começa a função **tipo de pessoa**.
https://github.com/deivisonarthur/OSC-Magento-Brasil-6-Pro/blob/master/app/design/frontend/base/default/template/onestepcheckout/onestep/form/address/billing.phtml

**Comment by casaud** [comment]
Acho que não é ali... 
Quando o cliente vai criar a conta, pessoa física, aparece o IE
Veja aqui => http://img.prntscr.com/img?url=http://i.imgur.com/msO9YFr.png

**Comment by casaud** [comment]
Achei uma possível solução...
Apagar o arquivo: https://github.com/deivisonarthur/OSC-Magento-Brasil-6-Pro/blob/master/app/design/frontend/base/default/template/onestepcheckout/customer/widget/ie.phtml

Porque no meu site é varejo mesmo... não vendo para empresas...

Pode apagar? Dá algum pau?

**Comment by rafaelstz** [comment]
aaa criar conta, ali é no checkout mesmo.

**Comment by rafaelstz** [comment]
linha 221 em diante, ele muda IE par RG (ou era pra fazer isso), talvez ele apenas não esteja identificando a label.
https://github.com/deivisonarthur/OSC-Magento-Brasil-6-Pro/blob/master/app/design/frontend/base/default/template/onestepcheckout/persistent/customer/form/register.phtml

**Comment by casaud** [comment]
Nem é isto... quero que retire da pessoa física o campo IE

**Comment by rafaelstz** [comment]
troca o **.first().update(** por show e hide nas funções de cada seleção, não esqueça de não deixar o RG/IE obrigatório.

**Comment by casaud** [comment]
Desculpe, mas não entendi direito... colocar show().update ? isto?

**Comment by rafaelstz** [comment]
Não, colocar como a linha 324 e 248 que exibe e oculta outro compo dependendo da seleção.

**Comment by casaud** [comment]
Não funcionou...

**Comment by rafaelobo** [comment]
casaud, conseguiu resolver isso?

**Comment by casaud** [comment]
Só consegui resolver apagando o arquivo ie.phtml.
Não queria assim, mas tá valendo.

**Comment by sidneydeus** [comment]
Ative o taxvat em configurações do cliente e altere o número de linhas de endereço para 4.
A verificação trava quando esses campos não existem.

**Comment by casaud** [comment]
Não entendi sidney... fiz aqui e ficou na mesma

**Comment by sidneydeus** [comment]
Teria que verificar algumas coisas como cache, conflito de js, voltar o phtml do ie...mas no meu caso foi resolvido assim numa instalação limpa.

**Comment by ismavolk** [comment]
Olá pessoal, pesquisando sobre inscrição estadual cheguei a este link que não tem muito haver com o que eu buscava, mas aproveitando que já estou aqui... Sim, pessoa física pode ter inscrição estadual, é o caso de produtores rurais por exemplo.

**Comment by DaviClaudino** [comment]
Alguém por acaso sabe como deixar o campo RG/IE obrigatório?
